### ISSUE TITLE
Feature Request: Specify Root Folder

### ISSUE THREAD

**Issue by jesterroyal** [carlonicora/obsidian-rpg-manager#39]
It would be nice to be able to specify the root folder for where the new notes and folders are automatically organized. Currently this seems to be the root of the Obsidian Vault or No auto-organization are the two options.

**Comment by carlonicora** [comment]
@jesterroyal if you place a campaign in a specific folder, all the elements created for that campaign are automatically moved in subfolders where the campaign lives.

**Comment by carlonicora** [comment]
@jesterroyal I have released a new version of RPG Manager - [v1.3](https://github.com/carlonicora/obsidian-rpg-manager/releases/tag/1.3.1)
This `should` have solved the issue you had (_I cannot test in Windows_). If you could test it with the new version, it would be amazing.

How it `should` work: RPG Manager creates subfolders starting from where your campaign sits. So if your campaign is in the folder `/My Campaign/My Campaign.md` the adventures should be moved into `/My Campaign/Adventures`

Thanks
Carlo

**Comment by jesterroyal** [comment]
@carlonicora  am unable to test this feature it seems. I do not have a pre-existing campaign and when attempting to either the "fill with campaign" or "create a new campaign" option i receive this error and no frontmatter is filled or no file is created.

```plugin:rpg-manager:4713 Uncaught (in promise) Error
    at Database.readSingleParametrised (plugin:rpg-manager:4713:13)
    at FileFactory.eval (plugin:rpg-manager:890:77)
    at Generator.next (<anonymous>)
    at eval (plugin:rpg-manager:43:61)
    at new Promise (<anonymous>)
    at __async (plugin:rpg-manager:27:10)
    at FileFactory.create (plugin:rpg-manager:887:12)
    at CampaignModal.eval (plugin:rpg-manager:980:65)
    at Generator.next (<anonymous>)
    at eval (plugin:rpg-manager:43:61)
````

**Comment by carlonicora** [comment]
@jesterroyal Thanks for the info. I think I know what it is and I will publish a patch soon

**Comment by carlonicora** [comment]
@jesterroyal fixed in [v1.3.2](https://github.com/carlonicora/obsidian-rpg-manager/releases/tag/1.3.2)

Would appreciate if you could let me know if both the automatic move works for you and also if you still have the issue creating the first campaign.

**Comment by jesterroyal** [comment]
Confirming that automatic move seems to work correctly for me and I am able to create a new campaign when no others exist.
